spec/win32/unit/store/assertions_spec.rb

```diff
@@ -17,6 +17,7 @@
 #
 
 require 'spec_helper'
+require 'win32/certstore/mixin/assertions'
```

Comment:
I understand that since these specs are executing for Linux too, so we need to require this file separately here as we'll have `require 'win32-certstore' if windows?` in the `spec_helper` file.